srcpkgs/discord-ptb/template

```diff
@@ -22,9 +22,9 @@ do_install() {
 	vmkdir usr/share/applications
 	vcopy discord-ptb.desktop /usr/share/applications/
 	vmkdir ${package_location}
-	for item in locales resources DiscordPTB libffmpeg.so libnode.so natives_blob.bin snapshot_blob.bin content_shell.pak icudtl.dat; do
+	for item in DiscordPTB chrome_100_percent.pak chrome_200_percent.pak icudtl.dat libEGL.so libGLESv2.so libffmpeg.so locales natives_blob.bin resources resources.pak snapshot_blob.bin swiftshader v8_context_snapshot.bin; do
```

Comment:
Preferably, split this into two lines using a `\` at the end of the line to make it more readable.

Reply:
Sure! I'm unsure on the formatting though. How long should the line be? Usually those are split on 90 characters.